packages/flutter/lib/src/widgets/focus_traversal.dart

```diff
@@ -1556,56 +1554,112 @@ class FocusTraversalGroup extends StatefulWidget {
   /// {@macro flutter.widgets.ProxyWidget.child}
   final Widget child;
 
-  /// Returns the focus policy set by the [FocusTraversalGroup] that most
-  /// tightly encloses the given [BuildContext].
+  /// Returns the [FocusTraversalPolicy] that applies to the nearest ancestor of
+  /// the given [FocusNode].
   ///
-  /// It does not create a rebuild dependency because changing the traversal
-  /// order doesn't change the widget tree, so nothing needs to be rebuilt as a
-  /// result of an order change.
+  /// Will return null if no [FocusTraversalPolicy] ancestor applies to the
+  /// given [FocusNode].
+  ///
+  /// The [FocusTraversalPolicy] is set by introducing a [FocusTraversalGroup]
+  /// into the widget tree, which will associate a policy with the focus tree
+  /// under the nearest ancestor [Focus] widget.
+  ///
+  /// This function differs from [maybeOf] in that it only traverses the focus
+  /// tree to determine the policy in effect. Unlike this function, the
+  /// [maybeOf] function first walks up the widget tree, to find the nearest
+  /// ancestor [Focus] or [FocusScope] widget, and then calls this function with
+  /// the focus node associated with that widget to determine the policy in
+  /// effect.
```

Comment:
Somewhere in this paragraph it would have been helpful for my own understanding to say that maybeOf takes a BuildContext (and walks the widget tree) while maybeOfNode takes a FocusNode (and walks the focus tree, which can have a different shape than the widget tree).